ddtrace/internal/packages.py

```diff
@@ -98,16 +98,19 @@ def _package_file_mapping():
         mapping = {}
 
         for ilmd_d in il_md.distributions():
-            if ilmd_d is not None and ilmd_d.files is not None:
-                d = Distribution(name=ilmd_d.metadata["name"], version=ilmd_d.version, path=None)
-                for f in ilmd_d.files:
-                    if _is_python_source_file(f):
-                        # mapping[fspath(f.locate())] = d
-                        _path = fspath(f.locate())
-                        mapping[_path] = d
-                        _realp = os.path.realpath(_path)
-                        if _realp != _path:
-                            mapping[_realp] = d
+            try:
+                if ilmd_d is not None and ilmd_d.files is not None:
+                    d = Distribution(name=ilmd_d.metadata["name"], version=ilmd_d.version, path=None)
+                    for f in ilmd_d.files:
+                        if _is_python_source_file(f):
+                            # mapping[fspath(f.locate())] = d
+                            _path = fspath(f.locate())
+                            mapping[_path] = d
+                            _realp = os.path.realpath(_path)
+                            if _realp != _path:
+                                mapping[_realp] = d
+            except ValueError:
+                continue
```

Comment:
Why is this necessary if the problem is with a package dependency?

Reply:
The exception is raised there and also we don´t want an error parsing one path to stop the full process in other cases.

Reply:
It was more for future-proofing in case importlib parsing errors break all the indexing loop, but maybe could be for another PR, removed.